packages/rs-drive-nodejs/src/lib.rs

```diff
@@ -100,8 +114,25 @@ impl PlatformWrapper {
                 ..Default::default()
             };
 
+            let core_config = CoreConfig {
+                rpc: CoreRpcConfig {
+                    url: core_rpc_url,
+                    username: core_rpc_username,
+                    password: core_rpc_password,
+                },
+            };
+
+            let platform_config = PlatformConfig {
+                drive: Some(drive_config),
+                core: core_config,
+            };
+
             // TODO: think how to pass this error to JS
-            let platform: Platform = Platform::open(path, Some(drive_config)).unwrap();
+            let mut platform: Platform = Platform::open(path, platform_config).unwrap();
+
+            if cfg!(feature = "enable-core-rpc-mocking") {
```

Comment:
I would recommend to rename feature to something more general because we gonna mock other things in future too. "enable-mocking"?